app/controllers/players/__init__.py

```diff
@@ -1,9 +1,17 @@
-from flask import Blueprint, jsonify
+from flask import Blueprint, jsonify, request
+from sqlalchemy import desc
 from app.models.player import Player
 
 players = Blueprint("players", __name__, url_prefix="/players")
 
 
 @players.route("/")
 def get_players():
-    return jsonify([player.as_dict() for player in Player.query.all()])
+    pos = request.args.get('position')
+
+    if pos:
+        return jsonify([player.as_dict()for player in Player.query.filter_by(
+            position=pos.upper()).order_by(desc('projection'))])
+    else:
+        return jsonify(
+            [player.as_dict() for player in Player.query.all()])
```

Comment:
A small potential change so that we only have 1 return statement in the method:

```suggestion
    if pos:
        players = Player.query.filter_by(
            position=pos.upper()).order_by(desc('projection'))])
    else:
        players = Player.query.all()])
    return jsonify([player.as_dict() for player in players])
```

Reply:
I like this change, I'll do it tomorrow.